climakitae/util/utils.py

```diff
@@ -520,3 +520,40 @@ def summary_table(data):
         df = df.sort_values(by=["year"])
 
     return df
+
+
+def convert_to_local_time(data, selections, lat, lon) -> xr.Dataset:
```

Comment:
Future iteration: pass in DataArray and location, function should figure it out from there.